src/app/submit-item/components/submit-success-page/submit-success-page.component.html

```diff
@@ -10,8 +10,7 @@
           <app-icon-with-content color="bittersweet" icon="thumbs-up">
             <span appIconWithContentHeader>Vielen Dank für's Einreichen!</span>
             <span>
-              Vielen Dank für das Einreichen dieses Falls. Die Community nimmt sich dem Fall an. Lorem ipsum dolor sit amet, consetetur
-              sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
+              Vielen Dank für das Einreichen dieses Falls. Die Community nimmt sich dem Fall an. Möchtest du nun selbst ein paar Fälle lösen? Weiter unten siehst du Fälle, bei denen die Community noch deine Hilfe benötigt.
```

Comment:
This phrase "nimmt sich dem Fall an" sounds a little bit outmoded. Sounds more correctly if a genitiv object is used ("nimmt sich des Falls an") even if the use with dativ object is allowed, too.

Would replace
"Vielen Dank für das Einreichen dieses Falls. Die Community nimmt sich dem Fall an. Möchtest du nun selbst ein paar Fälle lösen?"

with

"Vielen Dank für das Einreichen dieses Falls. Die Detektiv-Community kümmert sich jetzt darum! Möchtest du nun selbst ein paar Fälle lösen?"